app-office/worklog-assistant-bin/files/worklog-assistant-bin-2.1.3577-fix-desktop.patch

```diff
@@ -0,0 +1,11 @@
+--- "a/usr/share/applications/Worklog Assistant.desktop"	2016-11-19 15:59:42.000000000 +0100
++++ "b/usr/share/applications/Worklog Assistant.desktop"	2016-11-21 22:25:02.937080445 +0100
+@@ -3,7 +3,7 @@
+ [Desktop Entry]
+ Type=Application
+ Name=Worklog Assistant
+-Version=2.1.3577
++Version=1.0
```

Comment:
Since this is a kinda obvious point of WTF, the patch really needs a description on top. It would also probably be reasonable to move `Version` to top, to indicate it's really not referring to the program. Also submit this upstream, please ;-).